### PR TITLE
35 implement basic column masking

### LINKED ISSUE #35: implement basic column masking
implement basic star and hash column masking with the new architecture.
use the abac rego policy for attributes

### PR DESCRIPTION
**Pull request by TamKez**
Added row level filtering that works with the api.
The rule goes over the different attributes that are relevant in the user and table, and when the table requires rls, and the user's attribute is relevent for the tables content worls, rls policies are created.

### CODE REVIEW

File: opa/policies/cms/README.md [hunk 1]
```diff
@@ -1,7 +1,13 @@
+red {
+  color: red;
+}
+
 # CMS (Column Masking Security) Package
```
Comment (shohamyamin) [inline opa/policies/cms/README.md:5]: Add explanation about how the attribute affect column masking

File: opa/policies/rls/README.md [hunk 2]
```diff
@@ -19,3 +19,19 @@ To use this Rego package, follow these steps:
 To define row level security policies, you need to modify the `rls.rego` file. This file contains a set of rules that determine which rows are accessible to different users.
 
 Here's an example of how you can define a simple row level security policy:
+
```
Comment (shohamyamin) [inline opa/policies/rls/README.md:22]: Add explanation about how the row level attributes are used in the rls rules

File: opa/policies/rls/rls.rego [hunk 3]
```diff
@@ -1 +1,36 @@
...
+
+content_world_attributes := abac_am.content_world_attributes(table_attributes.attributes.content_world)
+
+filter[row_filter] {
+    table_attributes.attributes.row_filter == true
+    filter_feild := content_world_attributes.filter_key
+	user_attribute := user_attributes.attributes[_]
+    row_filtering_needed(user_attribute)
+	filter_value := user_attribute.filter_value[_]
+	expression := sprintf("%v <> %v", [filter_feild, filter_value])
```
Comment (shohamyamin) [inline opa/policies/rls/rls.rego]: Change the sql expression to be `nationKey in (1,2,3,4)`

change the misspelling feild to field

**Review by shohamyamin — CHANGES_REQUESTED** [review]
fix according to the feedback

**Review by shohamyamin — APPROVED** [review]

**Review by shohamyamin — APPROVED** [review]
looks good

**Review by shohamyamin — APPROVED** [review]
looks wonderful!!

**Review by nadavgross — APPROVED** [review]
nice

**Pull request merged**